pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -406,6 +406,24 @@ type MonitoringSpec struct {
 	// Interval determines prometheus scrape interval
 	// +optional
 	Interval *metav1.Duration `json:"interval,omitempty"`
+
+	// Ceph exporter configuration
+	// +optional
+	Exporter *CephExporterSpec `json:"exporter,omitempty"`
+}
+
+type CephExporterSpec struct {
+	// Only performance counters greater than or equal to this option are fetched
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=10
+	// +kubebuilder:default=5
+	PerfCountersPrioLimit uint `json:"perfCountersPrioLimit,omitempty"`
```

Comment:
Why maximum 10 here? I'll be honest that I don't have a good grasp of what this value represents, but I do see at least one instance a unit tests sets prio-limit=12. That suggests to me that 12 is a valid value. Ceph's config yaml doesn't list a maximum, and internally Ceph processes the priority as int64.

https://github.com/ceph/ceph/blob/ccdd877abaf707fabc7e0b8d209a421a2f841be0/src/exporter/DaemonMetricCollector.cc#L119-L121

I think it might be in our best interest to use the same data type as Ceph here (int64) and not set the maximum, unless you have deeper knowledge about what this value means that I'm missing.

FWIW, Ceph also doesn't limit this to positive integers. Can priorities be negative numbers? I don't know, but if there is doubt, it is better to not apply min/max values unless we are certain.

Reply:
If `prio` here represents linux kernel priorities, then it seems to me like -20 to 139 (seems kind of arbitrary) are valid values (from blog). But also seems like I am missing something since lower numbers are higher prio in most cases. With my own open questions, I would leave the min/max unset if I were working on this.

https://blogs.oracle.com/linux/post/task-priority